### ISSUE TITLE
Events module: feature requests

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#1706]
#### Issue

The Events module is a promising addition to SuiteCRM, but it lacks some crucial functionality for it to be useful to us. Here are a few additions to the events module that would make it useful to us:
- search/filter delegate (with hundreds of delegates there's currently no way to drill down to one particular one to change its status)
- export of delegates to target list (e.g. to send campaign emails with event details to all delegates that have accepted the invitation, or a thank-you email to all who attended, etc.)
- ability to sort delegate list by status
- (nice-to-have: possibility to narrow user-rights for delegate-editing. i.e. I'd like to allow everyone to look at an event record, including all delegate status details, without being able to edit anything in the event record and the delegate submodule. This is not important if issue [1704](https://github.com/salesagility/SuiteCRM/issues/1704) is resolved.)
#### Expected Behavior

see above
#### Context

These missing features make this module unusable for us, which is a shame, as we could make good use of it if it included just a few more features.
#### Your Environment

SuiteCRM 7.4.1

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Events module: feature requests](https://trello.com/c/6ekyDOB2/90-events-module-feature-requests)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
